### PR TITLE
ID-275 Try and add Travis support for id7 builds/tests

### PR DESCRIPTION
**Pull request by lol768**
Simple travis config that runs tests and tries to build docs.

### CODE REVIEW

**Comment by lol768** [comment]
image

They pass on the fork

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+matrix:
+  include:
+    - language: ruby
+      node_js: 
+        - lts
+      install:
+        - gem install bundler
+        - bundler install
+        - npm install --no-optional
```
Comment (halfninja) [inline .travis.yml]: Does `npm ci` work? If we don't do it on the Bamboo build we probably should

Comment (lol768) [reply .travis.yml]: image

**Comment by lol768** [comment]
Having thought about the FA Pro situation a bit, I don't think we want to the runner to have the token at all.

Since we'd be building on PR it seems far too easy to e.g. write a new JS test that exfils/leaks it which Travis would then happily run

**Comment by nanoamp** [comment]
Agreed. An alternative might be for us to stub an FA5 Pro analogue which would let us test the pipeline with a dummy token. Whether that's worth it is another matter...

**Comment by lol768** [comment]
Merging this first so I can make Travis use `npm ci` in the other PR

**Pull request merged**